conference.js

```diff
@@ -1916,21 +1912,12 @@ export default {
             JitsiE2ePingEvents.E2E_RTT_CHANGED,
             (...args) => APP.store.dispatch(e2eRttChanged(...args)));
 
-        APP.UI.addListener(UIEvents.AUDIO_MUTED, muted => {
-            this.muteAudio(muted);
-        });
-        APP.UI.addListener(UIEvents.VIDEO_MUTED, (muted, showUI = false) => {
-            this.muteVideo(muted, showUI);
-        });
-
         room.addCommandListener(this.commands.defaults.ETHERPAD,
             ({ value }) => {
                 APP.UI.initEtherpad(value);
             }
         );
 
-        APP.UI.addListener(UIEvents.EMAIL_CHANGED,
-            this.changeLocalEmail.bind(this));
```

Comment:
In conference.js this is the only code path that leads to `changeLocalEmail` being called. Since we are no longer doing that, can the function go ?

Reply:
It is used in other places. But We can actually convert it to redux action and move it somewhere else.

Reply:
Ah, I missed it the first time. Indeed, that can be done at a later time.